blockly-ws-search/README.md

```diff
@@ -0,0 +1,32 @@
+[Home](../README.md)
+
+# blockly-ws-search [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+This package provides an implementation of workspace search for a Blockly workspace.
+
+## Installation
+
+```
+npm install @blockly/plugin-workspace-search
+```
+
+## Usage
+
+```js
+var WorkSpaceSearch = require('@blockly/plugin-workspace-search');
```

Comment:
Default to ES6 imports here, especially that this isn't a package aimed at Node.js developers.

```md

```js


```

```